scripts/build_scripts/build_zips.py

```diff
@@ -562,8 +569,6 @@ def main(argv):
     cmake_setup_args.extend(get_ios_args(source_path))
   elif is_android_build():
     cmake_setup_args.extend(get_android_args())
```

Comment:
While Windows needs to be after the target path, is there any harm in these other platforms also being right after the target path?  That way the platform-specific if / else block is all in one place.